test/IRGen/Inputs/objc_direct.h

```diff
@@ -0,0 +1,7 @@
+#import <Foundation/Foundation.h>
+
+@interface Bar : NSObject
```

Comment:
Please also add a test for `__attribute__((objc_direct_members))`, and categories with direct methods.